### PR TITLE
[2] Add remaing builds.json files and fix projects

### PR DESCRIPTION
**Pull request by mchindri**
Will rebase onto #993 when merged

### CODE REVIEW

**Review by cristiantpop — APPROVED** [review]

**Comment by mchindri** [comment]
v2:
 - Fix astyle

**Comment by mchindri** [comment]
v3:
 - Add remaining projects.
 - Edit some commits titles

**Comment by mchindri** [comment]
All builds seems to run in almost 8 minutes :)

File: projects/adrv9009/builds.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                  "hardware": [
+                        "adrv9009_zc706",
+                        "adrv9009_zcu102"
+                  ]
+            },
+            "dma_example": {
+                  "flags" : "NEW_CFLAGS=-DDAC_DMA_EXAMPLE",
+                  "hardware": [
+                        "adrv9009_zc706",
+                        "adrv9009_zcu102"
```
Comment (buha) [inline projects/adrv9009/builds.json:14]: this project can run on more combinations of hardware but it's ok as-is, we can extend later on

File: projects/adrv9001/builds.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+      "xilinx": {
+            "demo": {
+                  "flags": "",
+                  "hardware": [
+                        "adrv9001_zc706",
+                        "adrv9001_zcu102",
+                        "adrv9001_zcu102_lvds",
+                        "adrv9001_zed"
```
Comment (buha) [inline projects/adrv9001/builds.json:9]: We do have HDL that has support for ZED + ADRV9001 but the no-OS project was never updated to support that.
zc706 is also with question mark.

Comment (mchindri) [reply projects/adrv9001/builds.json:9]: Will be update or should be removed from the build?

Comment (mchindri) [reply projects/adrv9001/builds.json]: removed zed and zc706 for the moment

**Pull request merged**